docs/UserGuide.md

```diff
@@ -2,16 +2,47 @@
 layout: page
 title: User Guide
 ---
+* Table of Contents 
+{:toc}
+--------------------------------------------------------------------------------------------------------------------
+## 1. Introduction
+### 1.1. About StoreMando
+Greetings, and welcome to StoreMando!
+
+Looking for an all-in-one solution for your inventory management needs? Look no further!
+
+
+StoreMando is an integrated platform fully customised for residents in households, residential colleges and halls, with 
+the aim of helping you manage your items effectively and efficiently. StoreMando allows you to keep track of your items' 
+whereabouts, quantities and expiry dates with a few simple commands. Furthermore, StoreMando also comes with an inbuilt 
+reminder feature to help you remember items that are expiring. It looks like you will never have to worry about consuming 
+expired items or waste time searching for misplaced items anymore!
+
+All your items' information is encapsulated clearly on our simple and clean Graphical User Interface (GUI). Our application 
+is optimised for use via the Command Line Interface (CLI) and if you have quick fingers, StoreMando can help you manage 
+your items in the blink of an eye.
+
+Explore this User Guide to find out more!
+
+### 1.2. Navigating the User Guide
 
-StoreMando is a **desktop app for managing inventory, optimized for use via a Command Line Interface** (CLI) while still
-having the benefits of a Graphical User Interface (GUI). If you can type fast, StoreMando can get your inventory
-management tasks done faster than traditional GUI apps.
+This user guide aims to provide you with all the information you need to make the most of StoreMando.
```

Comment:
Should it be 'make the most of StoreMando' or 'make the most out of StoreMando'

Reply:
I think both are fine but I'll change it to make it clearer.